packages/client/src/authentication/deleteTokens.ts

```diff
@@ -7,15 +7,15 @@ import join from 'proper-url-join';
  * @function deleteTokens
  * @memberof module:authentication/client
  *
- * @param {number} id - The user token id.
+ * @param {string} id - The user token id.
```

Comment:
Should we keep using these JSdoc types? I believe we can remove them as we type our code (also, I think we already have TSdoc configured and it might throw warnings for having these). What do you think?

Reply:
done. Changed to TSdoc on all files.